Makefile

```diff
@@ -228,9 +250,12 @@ test-docker-mount-code:
 
 lint:              		  ## Run code linter to check code style and check if formatter would make changes
 	($(VENV_RUN); python -m ruff check --show-source . && python -m black --check .)
+	$(VENV_RUN); pre-commit run upgrade-deps-if-changed --files $(git diff master --name-only)
```

Comment:
This make target should not test on changes, but perform the whole check. Is this the case here? Why is it the same as in `lint-modified`?

Reply:
The pre-commit hook is written in such a way that it needs to receive a list of files which it runs against. I can also just hardcode "setup.cfg" in the files option here instead.
What it does here currently is only run the check if the setup.cfg changed compared to the master branch.